publish/cli.go

```diff
@@ -213,6 +231,40 @@ func FXModuleFromFlags(cmd *cobra.Command, debug bool) fx.Option {
 			nats.MaxReconnects(maxReconnect),
 			nats.ReconnectWait(maxReconnectWait),
 		))
+	case sqsEnabled:
+		region, _ := cmd.Flags().GetString(iam.AWSRegionFlag)
+		sqsEndpointOverride, _ := cmd.Flags().GetString(SubscriberSqsEndpointOverrideFlag)
+
+		loadOptions := []func(*config.LoadOptions) error{
+			config.WithRegion(region),
+		}
+		if sqsEndpointOverride != "" {
+			// if we are overriding the endpoint assume we are in a dev context
+			loadOptions = append(loadOptions, config.WithCredentialsProvider(
+				credentials.NewStaticCredentialsProvider("dummy", "dummy", "dummy"),
+			))
+		}
+		cfg, err := config.LoadDefaultConfig(cmd.Context(), loadOptions...)
```

Comment:
I guess the config should be loaded in a Provide, or provided to the function.
It could avoid the panic.

Reply:
I was forgetting you could return errors in fx.Provide functions. Good idea. Done